src/classes/face.js

```diff
@@ -252,10 +253,25 @@ export class Face extends CircularLinkedList {
         return this;
     }
 
+    /**
+     * Merge two edges into one, edge_before will be extended,
+     * edge_after will be removed. The distortion of the polygon
+     * is on the responsibility of the use of this method
+     * @param {Edge} edge_before - edge to be extended
+     * @param {Edge} edge_after - edge to be removed
+     * @returns {Face}
+     */
+    merge_two_edges(edge_before, edge_after) {
```

Comment:
I'm confused why `edge_after` is an argument here and you don't use `edge_before.next`. (This method doesn't work well for any two edges, only incident ones.) Maybe I don't understand the separation between `Face` and `Polygon` though...